.github/workflows/nx-affected-test.yml

```diff
@@ -21,11 +23,11 @@ jobs:
                   cache: 'yarn'
             - run: git fetch origin main
             - run: yarn --immutable
-            - run: yarn affected:lint --parallel=4
+            - run: yarn affected:lint --parallel=8
```

Comment:
at some point I'd add this as a matrix option so that we can just change it in one location similar to how we have the node version defined.